src/collection-view.js

```diff
@@ -101,7 +101,7 @@ Marionette.CollectionView = Marionette.View.extend({
   _onCollectionAdd: function(child, collection, opts) {
     var index;
     if (opts.at !== undefined) {
-      index = opts.at;
+      index = opts.index || opts.at;
```

Comment:
As I wrote above:
> actually, when Backbone.Collection#add method called with a bunch of models, backbone triggers add event sequentially for each model, so where are no way to determine inside the _onCollectionAdd event handler whether one or many models are added

I think that there are no way to fix it for Backbone prior to 1.2.0
Replacing `at` with `index` will brake code for Backbone prior to 1.2.0
when `opts.index` is `0`, `opts.at` is `0` too...